sample_configuration/disco_aws.ini

```diff
@@ -32,9 +32,9 @@ default_product_line=teamname
 default_ami_available_wait_time=600
 
 [test]
-hostclass=mhcdiscointegrationtests
-command=/opt/wgen/disco_integration_tests/bin/run_tests.sh
-user=disco_tester
+test_hostclass=mhcdiscointegrationtests
```

Comment:
It'll work either way, but I assume we'll want to remove the the backwards compatibility code at some point, so easier to encourage people to use the new keys as soon as possible.